docs/supported-languages.md

```diff
@@ -214,11 +214,11 @@ This table provides information about the beta level of support for languages, s
   </tr>
   <tr>
    <td>Maven (single-file configurations)</td>
```

Comment:
Previously we only handled "single-file configurations" - how has this changed? @andy-r2c 🙏

Reply:
I'm not sure -- did we only handle single-file configurations previously for maven? cc: @mmcqd

Reply:
I believe this is referring to how we could not understand references within a `pom.xml` file to _other_ `pom.xml` files. I don't think this is a problem anymore. As long as a `maven_dep_tree.txt` file has been generated for every `pom.xml` the user wants us to understand, we're fine. Maven handles the hard part.

Reply:
Okay, I removed "single file configurations" now that our parser can handle all the things. :) TY!